leapp.spec

```diff
@@ -46,16 +46,20 @@ Requires:   libvirt-python
 Requires:   nmap
 Requires:   sshpass
 Requires:   python-enum34
+Requires:   docker 
+Requires:   rsync
```

Comment:
untill we have a proper failure message that docker has to be installed let's have it as required. regarding rsync i would prefer to have it required it's the only supported method to operate the product now.